README.md

```diff
@@ -364,15 +364,24 @@ scooby
 ```
 
 in a terminal will display the default report. You can also use it to show the
-scooby-report of another package, if that package has scooby implemented as
-suggested above, using `packagename.Report()`. As an example, to print the
-report of pyvista you can run
+scooby-report of another package: if that package has scooby implemented as
+suggested above, using `packagename.Report()`, otherwise it will get that
+packages dependences from its distribution to generate a report.
```

Comment:
"package: using scooby if the package has it implemented as suggested above, using `packagename.Report()`, or using its distribution to generate a report of its package dependences" (? wording solely)

Reply:
Yeah it was late at night when I wrote this... I'll iterate and make better